addon/-private/system/relationships/relationships-payloads.js

```diff
@@ -0,0 +1,417 @@
+import Ember from 'ember';
+let get = Ember.get;
+
+/**
+  Manages relationship payloads for a given store.  Acts as a single source of
```

Comment:
Can you expand on this comment to say that this is only until the relationship is initialized